### ISSUE TITLE
Remove reference to vagrant

### ISSUE THREAD

**Issue by marsbard** [marsbard/puppet-alfresco#78]
Installer contains reference to vagrant, remove it

**Comment by jpotts** [comment]
Is the reference that will be removed the one that says, "If you are running vagrant, press Q not I and run vagrant up"?

**Comment by marsbard** [comment]
Yes; I mostly removed the vagrant instructions just before I made the release, and I forgot that :-)

**Comment by marsbard** [comment]
also there is a reference to vagrant in https://github.com/marsbard/puppet-alfresco/blob/master/config/ootb_params.sh
